README.md

```diff
@@ -14,10 +14,10 @@ Check the online documentation at [https://momentobooth.github.io/momentobooth/]
 ## Features
 
 * Single capture
-* Multi-capture  
+* Multi-capture
   Shoot 4 photos and then select the ones you like to for a collage of 1, 2, 3, or 4 photos
 * User friendly touch-centered interface
-* Photo printing  
```

Comment:
Those spaces were there for a reason, it creates a line break. I can also accept an em-dash or something, but not this.

Reply:
It looks exactly the same in my VS Code preview. The newline is still there. What are you using for previewing this?

Reply:
GitHub, see [the preview of the readme for the branch](https://github.com/momentobooth/momentobooth/blob/1de93a7b30cf5e312fe6ac76a91e59603dfd7efc/README.md)

Reply:
Fixed!